gr-uhd/grc/CMakeLists.txt

```diff
@@ -53,6 +53,7 @@ if(ENABLE_UHD_RFNOC)
               uhd_rfnoc_addsub.block.yml
               uhd_rfnoc_fft.block.yml
               uhd_rfnoc_fosphor.block.yml
+              uhd_rfnoc_logpwr.block.yml
```

Comment:
This is never enforced, but considering #6301 and similar previous bugs, please keep things alphabetically sorted!